Dockerfile

```diff
@@ -1,49 +1,41 @@
 # syntax=docker/dockerfile:1
 
-# Stage 1: Build website
-FROM --platform=${BUILDPLATFORM} docker.io/library/node:24 AS website-builder
+# Stage 1: Build Node packages
+FROM --platform=${BUILDPLATFORM} docker.io/library/node:24-slim AS node-packages
 
-ENV NODE_ENV=production
-
-WORKDIR /work/website
+ARG GIT_BUILD_HASH
+ENV GIT_BUILD_HASH=$GIT_BUILD_HASH
 
-RUN --mount=type=bind,target=/work/website/package.json,src=./website/package.json \
```

Comment:
So the goal here is to only re-run this step when `package.json` or `package-lock.json` changed, and thus improve build speeds when only the code changes. I don't know how that would work with workspaces, probably need to mount a few more files, but in any case, we should preserve the current behavior

Reply:
I've only added this comment here, but it also goes for other changed dockerfiles